Record label-management transitions inline in the Orchestrator FSM (#471)
aunger/gallery-button-for-pixel-camera: agents/dev_orchestration.md

- See `inaugurate.md` for the full protocol when starting fresh work.

> ## Label management

Author: I think you, as the Agent writing these instructions, can greatly simplify the cyclometric complexity here. 

Wherever possible, evaluate the parentheticals and “signals” yourself. 
Match those to the branches (fences) already specified in the Orchestrator’s instructions.

Can you implement this with no conditions or signals inside a label-update table?

Extra credit: Can you implement this without adding any conditional *prose*?

Author: 🤖 Author

Done, in 757b8ce. Both the asked-for change and the extra credit.

Every label-update table is now unconditional: no parentheticals, no gating, and no "Signal" column anywhere inside a table. I evaluated those conditions at authoring time and matched each label move to the control-flow branch (fence) the Orchestrator already follows, so the surrounding branch (not the table) decides whether you are at a given table.

- Verifier transitions: the single three-column `Signal`-keyed table is gone. It is replaced by three unconditional two-column tables, one placed inside each of the three existing "Routing on the Verification Agent's signal" branches (`Verification passed`, `Verification revealed an error`, `Verification incomplete`).
- Reviewer returns: the unconditional `changes done` removal stays at the top of the Monitor loop (it happens on both LGTM and changes-requested). The `changes requested` add moves into the `goto newAuthor` branch, where the fence has already established that changes were requested. Neither table carries a condition.
- Verification Planner returns: the `verification needed` add moves into the existing non-empty ("Otherwise") branch. The empty branch just notes the Planner applies `verified` itself, so there is no table there.

Extra credit (no conditional prose): the conditional parentheticals in the tables are gone, and the explanatory notes that restated the conditions ("only if...", "depending on its answer", "when the list is not empty") are gone too. The only branch-selecting words left are the existing fence/branch headers that were already in the document before this label work. The "Label management" notation section no longer documents qualified remove cells, add-cell gating, or a "Signal" column; it now states plainly that every table is unconditional and is placed at the point where its move applies, and a `--` cell means nothing to remove or add.

markdownlint-cli2 and the rest of the pre-commit hooks pass.

---
_Generated by [Claude Code](https://claude.ai/code)_

After the Reviewer exits and delivers its decision, the Orchestrator acts as follows.
Monitor output lines are relayed to the user verbatim; this is user-facing status reporting and is not governed by the say-nothing rule (which covers sub-agent messages only).

When the Reviewer returns, apply this transition to the PR before branching on its decision:

| Remove label | Add label |
|---|---|
| `changes done` | -- |

Then branch on the Reviewer's decision.

> In the `goto newAuthor` branch below (the Reviewer requested changes), also apply this transition to the PR:

Author: Simplify here. Just say this is a conditional.

```suggestion
If the Reviewer requested changes, apply this transition to the PR:
```

Author: 🤖 Author

Done in b8ec65a. The lead-in now reads `If the Reviewer requested changes, apply this transition to the PR:`, dropping the `goto newAuthor` cross-reference. The table beneath it is unchanged.

---
_Generated by [Claude Code](https://claude.ai/code)_

      Dispatch a Verification Planner sub-agent using the dispatch template.
      The planner assembles the before-merging list and, for each item, files a tracking issue linked to the PR (see verification_planning.md). It does not consult the user.
      If the Planner reports its before-merging list is empty, then this *before-merging requirements* process is complete, and the Orchestrator should exit this step.

>       (The Planner itself applies `verified` to both the PR and the issue in this case, so the Orchestrator has no label move to make.)

Author: I forgot about this. Please attempt to simplify everything by moving the labeling jobs all to the Orchestrator.

Author: 🤖 Author

Done in 46b280d. All label moves now live with the Orchestrator.

The only labeling job that was not already the Orchestrator's was the Verification Planner applying `verified` when its before-merging list came back empty. I moved that:

- `agents/verification_planning.md` step 2 no longer applies `verified`; the Planner just reports the empty list. The note now says the Orchestrator owns all label moves and applies `verified` on that report.
- `agents/dev_orchestration.md`: the empty-before-merging branch now carries a `-- | verified` table applied to both the issue and the PR (replacing the parenthetical that said the Planner did it), and the reference index points at both before-merging branches.

Every other transition (start, Author returns, Reviewer returns, Verifier routing, conclude) was already an Orchestrator table, so labeling is now fully consolidated under the Orchestrator.

---
_Generated by [Claude Code](https://claude.ai/code)_
